requirements.txt

```diff
@@ -1,4 +1,4 @@
-taurus-citrine==0.4.0
+taurus-citrine==0.4.1
```

Comment:
Do we want 0.4.1 or 0.5.0?

Reply:
We never actually re-applied the update to 0.5.0.